lib/PuppeteerSharp.Tests/Page/GotoTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class GotoTests : PuppeteerBaseTest
+    {
+        [Fact]
+        public async Task ShouldNavigateToAboutBlank()
+        {
+            using (var page = await Browser.NewPageAsync())
+            {
+                var response = await page.GoToAsync(TestConstants.AboutBlank);
+                Assert.Null(response);
+            }
+        }
+
+        [Theory]
+        [InlineData("domcontentloaded")]
+        [InlineData("networkidle0")]
+        [InlineData("networkidle2")]
```

Comment:
What do you think about making a class in Puppeteer with these constants?
Something like
```cs
public class NativationEvents
{
    public const DOMContentLoaded = "domcontentloaded";
    ...
    ...
}
```

Reply:
An enum sounds better since it's more user friendly

Reply:
this will remove the test `ShouldThrowIfNetworkidleIsPassedAsAnOption` 👍